### ISSUE TITLE
action logs are out of order

### ISSUE THREAD

**Issue by jmhodges** [Manuel-777/MTG-Arena-Tool#123]
The action logs are adding events out of order. See below. By line 3, the Nordicminotaur is said to have played a Mountain on my turn (#2). This continues on for the rest of the game.

```
23:39:12 Nordicminotaur's turn begin. (#1)
23:39:13 jmhodges's turn begin. (#2)
23:39:13 Nordicminotaur played Mountain
23:39:13 jmhodges drew a card
23:39:17 Nordicminotaur's turn begin. (#3)
23:39:17 jmhodges played Island
23:39:17 Nordicminotaur drew a card
23:39:21 Nordicminotaur played Island
23:39:22 jmhodges's turn begin. (#4)
23:39:22 Nordicminotaur casted Goblin Electromancer
23:39:22 Nordicminotaur resolved Goblin Electromancer
23:39:22 jmhodges drew a card
23:39:27 Nordicminotaur's turn begin. (#5)
```

**Comment by jmhodges** [comment]
(This is with the currently newest mtg arena and v2.2.2 version of this tool)

**Comment by Manuel-777** [comment]
If you could send an output log of this one it would be great, I have seen reports but never managed to "catch it" in a log file.

**Comment by jmhodges** [comment]
It looks like my output_log.txt got overwritten in the last day. Shoot. I'll check in on future games and grab it before the process restarts

**Comment by Manuel-777** [comment]
Yes, output_log is reset whenever Arena is started, beware of that :)

**Comment by jmhodges** [comment]
Okay, it was definitely out of order in the "Sleepyj" game. There's a bunch more games in here and it's the last one.

[output_log.zip](https://github.com/Manuel-777/MTG-Arena-Tool/files/2682492/output_log.zip)

**Comment by jmhodges** [comment]
This is still marked as needs more data. I've put up an output log zip file. Do you need more than that?
